### PR TITLE
JBIDE-26444 Add deploy on server in Central ITests

### PR DESCRIPTION
**Pull request by jkopriva**
Signed-off-by: Josef Kopriva <jkopriva@redhat.com>

### CODE REVIEW

**Comment by jkopriva** [comment]
@odockal can you please review? Build on Jenklins is failing due to javascript issues in central.

File: tests/org.jboss.tools.central.ui.bot.test/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -16,7 +16,9 @@ Require-Bundle: org.eclipse.ui.forms;bundle-version="3.5.101",
  org.eclipse.ui;bundle-version="3.107.0",
  org.apache.commons.lang;bundle-version="2.6.0",
  org.jboss.tools.common.reddeer;bundle-version="3.10.0",
- org.jboss.tools.maven.reddeer
+ org.jboss.tools.maven.reddeer,
+ org.eclipse.reddeer.requirements,
```
[the remark this answers is not in the document]
Comment (jkopriva) [reply tests/org.jboss.tools.central.ui.bot.test/META-INF/MANIFEST.MF]: thx, done :)

File: tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/HTML5Parameterized.java [hunk 2]
```diff
@@ -237,5 +266,43 @@ private String getProjectName() {
...
+
+	private void checkDeployedProject(String projectName) {
+		ServersView2 serversView = new ServersView2();
+		serversView.open();
+		ServerModule module = serversView.getServer(FULL_SERVER_NAME)
+				.getModule(new RegexMatcher(".*" + projectName + ".*"));
+		new WaitUntil(new ServerModuleHasState(module, ServerState.STARTED), TimePeriod.getCustom(30));
+		ModuleLabel moduleLabel = module.getLabel();
+		ServerState moduleState = moduleLabel.getState();
+		assertTrue("Module has not been started!", moduleState == ServerState.STARTED);
```
[the remark this answers is not in the document]
Comment (jkopriva) [reply tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/HTML5Parameterized.java]: thx, done :)

**Comment by odockal** [comment]
@jkopriva Seems alright, you get my +1. But you must merge it yourself ;)

**Comment by jkopriva** [comment]
@odockal thx :)

**Pull request merged**